load-testing/tests/appGateServerEtherium.js

```diff
@@ -0,0 +1,25 @@
+// Our custom handleSummary produces HTML report with CLI output and some default configuration.
```

Comment:
`app_gate_server.js`?

That way we can add logic in the same place.

Reply:
There will be different tests for app gate server, not just one.